README.md

```
@@ -16,38 +18,107 @@ It's impossible to talk about *"mockingbird"* without remembering the song [Mock
-```
 
-- configure your shadow cljs:
+$ npx create-cljs-project your-project
+$ npm install react autoprefixer babel-loader css-loader cssnano karma karma-chrome-launcher karma-cljs-test msw npm-run-all postcss postcss-cli postcss-loade postcss-preset-env shadow-cljs tailwindcss webpack webpack-cli rimraf glob @isaacs/cliui --save-dev
+
+```
 
-> for better understanding, follow the offcial [guide](https://github.com/thheller/shadow-cljs)
+Copy our [tailwind.config.js](tailwind.config.js), especially the colors, into your own config.
 
+Consider creating a package.json file with these dependencies pre-configured or, maybe, you can even copy our package.json. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)